README.md

```diff
@@ -322,6 +322,70 @@ cd src/wasm-lib
 cargo test
 ```
 
+### Mapping CI CD jobs to local commands
+
+When you see the CI CD fail on jobs you may wonder three things 
+- Do I have a bug in my code? 
+- Is the test flaky? 
+- Is there a bug in `main`?
+
+To answer these questions the following commands will give you confidence to locate the issue.
+
+#### Static Analysis
+
+Part of the CI CD pipeline performs static analysis on the code. Use the following commands to mimic the CI CD jobs.
+
+The following set of commands should get us closer to one and done commands to instantly retest issues.
+
+```
+yarn test-setup
+```
+
+> Gotcha, are packages up to date and is the wasm built?
+
+
+```
+yarn tsc
+yarn fmt-check
+yarn lint
+yarn xstate:typegen
+yarn test:unit:local
+```
+
+> Gotcha: Our unit tests have integration tests in them. You need to run a localhost server to run the unit tests.
+
+#### Integration Tests
```

Comment:
We use the end to end/e2e nomenclature for playwright, could we stick to that?
```suggestion
#### E2E Tests
```

I think integration tests, test more than one system working together, but not the entire stack, so `src/lang/modifyAst/addFillet.test.ts` would 100% count as an integration test, but not a e2e test.